wordle-web/src/views/Game.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <main>
+    <v-card class="mx-auto">
+      <v-card-item>
```

Comment:
`<v-card-item>` not necessary. some output could be gotten without the extra layer